dist/overwolf/manifest.json

```diff
@@ -13,8 +13,11 @@
     "launcher_icon": "icons/desktopIcon.ico",
     "window_icon": "icons/taskbarIcon.png"
   },
-  "permissions": ["VideoCaptureSettings"],
+  "permissions": ["VideoCaptureSettings", "Extensions", "Streaming", "Profile", "GameInfo"],
```

Comment:
Is this required for the ads? (not sure)

Reply:
for display ads, this permissions are not required, they recommend them to enhance tracking and better results. We should decide together, if we want that for our app, especially because it transfers data to google

Reply:
Left them in for now, will bring up the issue at noon.